tests/vmi_lifecycle_test.go

```diff
@@ -1614,12 +1614,14 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Signaled graceful shutdown for %s", vmi.GetObjectMeta().GetName())), "Should log graceful shutdown")
 
 				// Verify VirtualMachineInstance is killed after grace period expires
-				By("Checking that the VirtualMachineInstance does not exist after grace period")
-				Eventually(func() string {
-					data, err := logsQuery.DoRaw(context.Background())
-					Expect(err).ToNot(HaveOccurred(), "Should get logs")
-					return string(data)
-				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log graceful kill")
+				if !libvmifact.IsS390X() { //Exclude this for the S390x architecture, as graceful shutdown is sufficient.
+					By("Checking that the VirtualMachineInstance does not exist after grace period")
+					Eventually(func() string {
+						data, err := logsQuery.DoRaw(context.Background())
+						Expect(err).ToNot(HaveOccurred(), "Should get logs")
+						return string(data)
+					}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log graceful kill")
+				}
```

Comment:
```suggestion
				if libvmifact.IsS390X() { 
					//Exclude this for the S390x architecture, as graceful shutdown is sufficient.
					return
				}
				By("Checking that the VirtualMachineInstance does not exist after grace period")
				Eventually(func() string {
					data, err := logsQuery.DoRaw(context.Background())
					Expect(err).ToNot(HaveOccurred(), "Should get logs")
					return string(data)
				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log graceful kill")
```

I would use an early return here, instead of putting the whole codeblock into the if.

Reply:
Yes, but I think it's better to keep it within the block. If additional validations are added to the test in the future, this approach ensures that only this specific validation is skipped. This way, we avoid unintentionally skipping other validations